kernel/examples/read-table-multi-threaded/src/main.rs

```diff
@@ -105,19 +106,13 @@ fn truncate_batch(batch: RecordBatch, rows: usize) -> RecordBatch {
 }
 
 // This is the callback that will be called fo each valid scan row
-fn send_scan_file(
-    scan_tx: &mut spmc::Sender<ScanFile>,
-    path: &str,
-    size: i64,
-    _stats: Option<Stats>,
-    dv_info: DvInfo,
-    partition_values: HashMap<String, String>,
-) {
+fn send_scan_file(scan_tx: &mut spmc::Sender<ScanFile>, file: delta_kernel::scan::state::ScanFile) {
```

Comment:
Likewise here, no need to unwrap and then re-wrap the struct, just use the kernel one